src/common/Card/style.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+
+export const StyledCard = styled.div`
+  background-color: white;
+  border: 1px solid #dbdbdb;
+  border-radius: 8px;
+  overflow: hidden;
+  @media only screen and (max-width: 1200px) {
+    background-color: white;
+    border: 1px solid #dbdbdb;
+    border-radius: 8px;
+    overflow: hidden;
+    margin: 0.5em;
+    min-width: max-content;
+  }
+`;
+
+export const EventImgContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: 145px;
+  background-color: #f7ffe1;
+  overflow: hidden;
+`;
+
+export const EventImg = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
+
+export const EventInfo = styled.div`
+  padding: 0.75em;
+`;
+
+export const EventTitle = styled.div`
+  font-family: "Roboto Mono", monospace;
+  font-weight: bold;
+  font-size: 1.1em;
+`;
+
+export const EventTime = styled.div`
```

Comment:
This should be `styled.p` instead of a `styled.div` (using semantic HTML etc). Also, I don't think we should specify the font names here as they will be specified by our scss.